support-frontend/assets/components/checkoutBenefits/checkoutBenefitsList.tsx

```diff
@@ -79,10 +90,14 @@ export function CheckoutBenefitsList({
 		>
 			<h2
 				css={
-					withBackground ? [headingCss, maxWidth] : [headingCss, smallMaxWidth]
+					withBackground
+						? displayEmotionalBenefit
+							? [headingCss]
+							: [headingCss, maxWidth]
+						: [headingCss, smallMaxWidth]
```

Comment:
Can we separate out these conditions into variables so that we can avoid the double ternary